build.override.targets

```diff
@@ -3,37 +3,49 @@
 
   <PropertyGroup>
     <TraversalBuildDependsOn>
+      WcfCopyProjectJson;
       WcfSetup;
       $(TraversalBuildDependsOn);
       WcfCleanup;
     </TraversalBuildDependsOn>
   </PropertyGroup>
-  
+
   <PropertyGroup>
     <TraversalCleanDependsOn>
       WcfClean;
       $(TraversalCleanDependsOn);
     </TraversalCleanDependsOn>
   </PropertyGroup>
-  
+
   <PropertyGroup>
     <WcfSelfHostSetupLog>$(MSBuildThisFileDirectory)\SelfHostedWcfServiceSetup.log</WcfSelfHostSetupLog>
     <WcfSelfHostCleanupLog>$(MSBuildThisFileDirectory)\SelfHostedWcfServiceCleanup.log</WcfSelfHostCleanupLog>
   </PropertyGroup>
-  
+
+  <PropertyGroup>
+    <UseLatestDependencies Condition="'$(UseLatestDependencies)'==''">false</UseLatestDependencies>
+  </PropertyGroup>
```

Comment:
Clarified - we're relying on the default value below in line 34 to set if no pre-conditions such as "UseLatestDependencies" are true. There may be other conditions we will add later